pkg/container-collection/containers.go

```diff
@@ -22,15 +22,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/rest"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 )
 
 // Container represents a container with its metadata.
 type Container struct {
+	// Container Runtime
+	Runtime string `json:"runtime,omitempty" column:"runtime,minWidth:5,maxWidth:10" columnTags:"runtime"`
+
 	// ID is the container id, typically a 64 hexadecimal string
```

Comment:
These could also be added as `columnDesc` later on.

Reply:
Agree. I think that's one of the next features we need to work on: Show the available (and description) columns in `--help`.

Reply:
Issue for ^ https://github.com/kinvolk/inspektor-gadget/issues/998.